cmd/security-spire-config/Dockerfile

```diff
@@ -0,0 +1,63 @@
+#  ----------------------------------------------------------------------------------
+#  Copyright 2022 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+# 
+#  ----------------------------------------------------------------------------------
+
+# Build utility container
+ARG BUILDER_BASE=golang:1.17-alpine3.15
+FROM ${BUILDER_BASE} AS builder
+
+WORKDIR /edgex-go
+
+RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+
+RUN apk add --update --no-cache make git
+
+COPY go.mod vendor* ./
+RUN [ ! -d "vendor" ] && go mod download all || echo "skipping..."
+
+COPY . .
+
+# The main mirrors are giving us timeout issues on builds periodically.
+# So we can try these.
+#RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+#RUN apk add --update --no-cache zeromq-dev libsodium-dev pkgconfig build-base git
+
+ARG SPIRE_RELEASE=1.2.0
+
+RUN wget -q "https://github.com/spiffe/spire/releases/download/v${SPIRE_RELEASE}/spire-${SPIRE_RELEASE}-linux-x86_64-glibc.tar.gz" && \
+    tar xv --strip-components=1 -f "spire-${SPIRE_RELEASE}-linux-x86_64-glibc.tar.gz"
+
+# Deployment image
+#FROM debian:stable-slim
+FROM alpine:3.15
+
+LABEL license='SPDX-License-Identifier: Apache-2.0' \
+      copyright='Copyright (c) 2022 Intel Corporation'
+
+#ENV DEBIAN_FRONTEND=noninteractive
+#RUN apt-get update -y && apt-get upgrade -y && apt-get install -y --no-install-recommends dumb-init && apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apk update && apk --no-cache --update add dumb-init gcompat
```

Comment:
See prior comments about removing the commented blocks now, and repeating the `sed` line prior to apk install for the alpine container.